features/admin.feature

```diff
@@ -226,7 +226,8 @@ Feature:  Administration
     Then I should see "search.gov/oops"
     And I follow "Fetch"
     And I wait for ajax
-    Then I should see "404"
+    Then I should see "Your URL has been added to the fetching queue"
+    And I follow "Close" within "div.info-message.message"
```

Comment:
@ethanewing , remember to keep cucumber features as 'human-readable' as possible. For `within` steps, you'll need to define one a 'human-readable' page area. See https://github.com/GSA/search-gov/blob/master/features/step_definitions/within_steps.rb

In this case, you might say something like `within the first URLs row`. I suspect that `div.info-message.message` isn't the best CSS id for that; let me know if you need help finding a better identifier.